app/config/config_dev.yml

```diff
@@ -14,15 +14,18 @@ web_profiler:
 monolog:
     handlers:
         main:
-            type:  stream
-            path:  %kernel.logs_dir%/%kernel.environment%.log
-            level: debug
+            type:   stream
+            path:   %kernel.logs_dir%/%kernel.environment%.log
+            level:  debug
+        console:
+            type:   console
+            bubble: false
```

Comment:
It's all about Monolog configuration, so IMO the dev should read the docs and know that the order is important when defining log handlers as said in the [core concepts](https://github.com/Seldaek/monolog/blob/master/doc/01-usage.md#core-concepts) (dead link abofe)